src/app/content/highlights/reducer.spec.ts

```diff
@@ -33,11 +33,12 @@ describe('highlight reducer', () => {
     expect(state.currentPage.highlights).toEqual(initialState.currentPage.highlights);
   });
 
-  it('locationChange - reset hasUnsavedHighlight', () => {
+  it('locationChange - reset hasUnsavedHighlight and focused', () => {
     const state = reducer(
-      {...initialState, currentPage: {...initialState.currentPage, hasUnsavedHighlight: true}},
+      {...initialState, currentPage: {...initialState.currentPage, hasUnsavedHighlight: true, focused: 'asd'}},
```

Comment:
I think this doesn't doesn't cover the bug scenario. It needs the `pageUId` to be set to `asdf` so that it goes to the `omit('focused', {...state.currentPage, hasUnsavedHighlight: false})` that was added as a fix.